closed/autoconf/custom-hook.m4

```diff
@@ -464,7 +464,7 @@ AC_DEFUN([OPENJ9_THIRD_PARTY_REQUIREMENTS],
     fi
 
     if test "x$OPENJDK_BUILD_OS_ENV" = xwindows.cygwin ; then
-      FREEMARKER_JAR=`$CYGPATH -m "$with_freemarker_jar"`
+      FREEMARKER_JAR=`cygpath -m "$with_freemarker_jar"`
```

Comment:
Is there a problem with `$CYGPATH`?

Reply:
Yeah, `CYGPATH` is empty when the command in question is executed.

Reply:
```
20:46:53  ./common/omrcuda.cpp(41): fatal error C1083: Cannot open include file: 'cuda.h': No such file or directory
```
Will test `cuda`.

Reply:
I think we should try to order things so this can continue to use `$CYGPATH`.

Reply:
It appears that `CYGPATH` is superseded by `PATHTOOL`: see [basic_tools.m4](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9-staging/make/autoconf/basic_tools.m4#L87).

Reply:
On the other hand, `OpenJDK` uses `cygpath` extensively from searching result.

Reply:
There aren't so many references in the `openj9-staging` branch; most have been replaced by `PATHTOOL`. We should do likewise.